dpnp/dpnp_utils.pyx

```diff
@@ -96,27 +96,7 @@ def call_origin(function, *args, **kwargs):
     return result
 
 
-def check_attr_equal(func_name, attr_name, *input_arrays):
-    """
-    Check value of the specified attribute of all input arrays is equal,
-    otherwise ValueError is rased.
-    
-    Parameters
-    ----------
-    func_name : str
-        name of the function to raise the exception
-    attr_name : str
-        name of the attribute to check
-    input_arrays : tuple(arrays)
-        input arrays
-    """
-    unique_vals = set(getattr(x, attr_name) for x in input_arrays)
-    if len(unique_vals) > 1:
-        x1, x2, *_ = list(unique_vals)
-        checker_throw_value_error(func_name, attr_name, x1, x2)
-
-
-def check_nd_call(origin_func, dpnp_func, supported_dtypes, *input_arrays,
+def check_nd_call(origin_func, dpnp_func, *input_arrays,
```

Comment:
Could you please move this function to dpnp_iface_bitwise.py?
As far as I understand this function can not be simply used in other modules. This function is strictly tied with `bitwise` functionality.